fgcm/fgcmUtilities.py

```diff
@@ -471,11 +471,12 @@ def plotCCDMap(ax, deltaMapper, values, cbLabel, loHi=None):
     ax.scatter(deltaMapper['delta_ra'].ravel(), deltaMapper['delta_dec'].ravel(),
                s=0.1,
                c=zGrid.ravel(),
-               vmin=lo, vmax=hi)
+               vmin=lo, vmax=hi,
+               cmap=cm)
     ax.set_aspect('equal')
 
     cb = None
-    cb = plt.colorbar(CS3, ticks=np.linspace(lo, hi, 5), ax=ax)
+    cb = plt.colorbar(CS3, ticks=np.linspace(lo, hi, 5), ax=ax, cmap=cm)
```

Comment:
I think you could use your `scalarMap` from above here (`plt.colorbar(scalarMap)`) , then not need to set anything else, but I'm not 100% sure what you're trying to do.